package.json

```diff
@@ -18,12 +18,13 @@
   "dependencies": {
     "body-parser": "^1.10.1",
     "cors": "^2.5.2",
+    "debug": "^2.2.0",
```

Comment:
As far as I can tell, we're not even using this dependency. Can we just get rid of it?